### ISSUE TITLE
udev input configuration broken on particular HW configuration in X session

### ISSUE THREAD

**Issue by asbytes** [systemd/systemd#14822]
**systemd version the issue has been seen with**
> 244.2-1-arch

**Used distribution**
 > Arch Linux

**Expected behaviour you didn't see**
 > Mouse and keyboard connected to laptop through USB-hub must work in X session

**Unexpected behaviour you saw**
 > Mouse and keyboard do not work in X session, in terminal session all works

**Steps to reproduce the problem**
Hardware setup is laptop with mouse and keyboard connected through USB hub. Model of hub is Defender Septima Slim. Problem appeared after commit 95f2b4dd237faa57fd3e93245d560e47cdedfc2c. 
Hub info:
```
$ lsusb
ID 1a40:0201 Terminus Technology Inc. FE 2.1 7-port Hub
```
It has the same VendorId/ProductId as in udev rule, introduced in commit above
```
# 'Plugable UD-PRO8' USB hub, sound, network, graphics adapter
SUBSYSTEM=="usb", ATTR{idVendor}=="1a40", ATTR{idProduct}=="0201", ENV{ID_AUTOSEAT}="1"
```
Xorg has single seat configuration, so when udev detects hub and applies auto-seat property to environment, devices behind hub become not visible by X session. After commenting line with this rule all works fine.

**Comment by keszybz** [comment]
I pushed the commit to the stable branches, and clearly this wasn't a good idea. I'll revert it there. Not sure what to do in the master branch... @ezst036 ?

**Comment by ezst036** [comment]
Can we set the environment variable ENV{ID_AUTOSEAT} to zero?

# 'Plugable UD-PRO8' USB hub, sound, network, graphics adapter
SUBSYSTEM=="usb", ATTR{idVendor}=="1a40", ATTR{idProduct}=="0201", ENV{ID_AUTOSEAT}="0"

That should make it visible, but it won't(or shouldn't) engage the auto-configure logic.  I should handle enabling these seat devices in the UI instead.

**Comment by hpfmn** [comment]
Same issue here - setting END{ID_AUTOSEAT}="0" does indeed work.

**Comment by asbytes** [comment]
There was an issue with the same behavior, which was solved in d7e1c95e0a22eb1ada8ad53dab3af5fec2af1d0c. As i see, additional attributes of device are provided to specify it from generic usb hub with the same vendor/product. The string 
```
ATTR{%k.2/idVendor}=="17e9", ATTR{%k.2/idProduct}=="401a", ATTR{%k.2/product}=="mimo inc"
``` 
in _# Match parent_ rule for mimo 720 device specifies usb hub by its child device, as i understand.
@ezst036 Maybe you could find such values for your equipment and add them to rule?

**Comment by hussamT** [comment]
This may be offtopic but how is 244.3 tagged but the commit is not showing in https://github.com/systemd/systemd-stable/commits/v244-stable?

**Comment by eworm-de** [comment]
I fixed this, pushed v244.3 to systemd/systemd-stable.

**Comment by keszybz** [comment]
This has been reverted in 99fdffaa194cbfed659b0c1bfd0ace4bfcd2a245. A new PR will be needed to restore the functionality.

/cc @ezst036
